pdns/recursordist/docs/lua-scripting/hooks.rst

```diff
@@ -272,13 +272,14 @@ Example script
     myDomain = newDN("example.com")
 
     myNetblock = newNMG()
-    myNetblock:addMasks("192.0.2.0/24")
+    myNetblock:addMasks({"192.0.2.0/24"})
```

Comment:
Just a nit but `myNetblock:addMask("192.0.2.0/24")` should also work.

Reply:
ok - but it is somewhat of a wash if replacing it will improve things now, do you agree?

Reply:
Sure!